app.js

```diff
@@ -0,0 +1,60 @@
+var express = require('express');
```

Comment:
Is there a reason why you made an express app? I can't tell if its doing anything. 
I think most of the exercises can be done in just the node runtime.